spec/models/user_spec.rb

```diff
@@ -10,4 +10,62 @@
 
     expect(user.interests.count).to eq 2
   end
+
+  describe '#to_csv' do
+    it 'returns user interests list' do
```

Comment:
There should be a case for when user actually have some interests. And it returns more than just "interests list"